plenum/server/validator_info_tool.py

```diff
@@ -459,11 +459,10 @@ def _get_node_metrics(self):
 
     def _get_ic_queue(self):
         ic_queue = {}
-        for view_no, queue in self._node.view_changer.instanceChanges.items():
-            ics = {}
-            ics["Vouters"] = self._prepare_for_json(list(queue.voters))
-            ics["Message"] = self._prepare_for_json(queue.msg)
-            ic_queue[view_no] = self._prepare_for_json(ics)
+        for view_no, votes in self._node.view_changer.instance_changes.items():
```

Comment:
Are we changing the validator info format here?

Reply:
Yes, because we are not store an InstanceChange message now. But for validator info we can return the old structure if this critical. 
A new structure a bit better because we show a reason to change view for all voters.